.env.example

```diff
@@ -1,3 +0,0 @@
-REACT_APP_CLIENT_ID=the client id obtained from Google Developer console.
-REACT_APP_BACKEND_ENDPOINT=http://localhost:8080
-REACT_APP_BACKEND_SOCKETIO=http://localhost:8081
```

Comment:
Don't think this file should be removed for future purposes. This file should remain in the repository as a template file in case anyone else wants to set this up from the start.